rm_vk/rm_vk/NewsFeedViewController/ViewController/NewsFeedViewController.swift

```diff
@@ -0,0 +1,128 @@
+// NewsFeedViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Гланый экран новостей
+final class NewsFeedViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let headerTableViewCellIdentifier = "HeaderTableViewCell"
+        static let postTextTableViewCellIdentifier = "PostTextTableViewCell"
+        static let postPhotoTableViewCellIdentifier = "PostPhotoTableViewCell"
+        static let footerTableViewCellIdentifier = "FooterTableViewCell"
+    }
+
+    // MARK: - IBOutlet
+
+    @IBOutlet private var tableView: UITableView!
+
+    // MARK: - Private Properties
+
+    private let networkService: NetworkServiceProtocol = NetworkService()
+    private var items: [CellType] = []
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchNews()
+    }
+
+    // MARK: - Private Methods
+
+    private func fetchNews() {
+        networkService.fetchNews { result in
```

Comment:
```suggestion
        networkService.fetchNews {   [weak self] result in
```
 пока вы спорили про алерты, проглядели  утечку памяти

Reply:
я саше писала в личку нужно здесь или нет) а он у меня не заметил)